### PR TITLE
Update local-groups.mdx

### PR DESCRIPTION
**Pull request by Goyath**
add Japan

### CODE REVIEW

**Comment by Goyath** [comment]
Do I need to create an account to access Vercel?

**Comment by tekstrand** [comment]
Hi @Goyath thanks for opening a PR to add the Japan group! No need to get Vercel.

You will need to change the name of the group to something like Japan Meshtastic Club, Japan Meshtastic Community, or something of that sort to comply with the [Licensing & Trademark](https://meshtastic.org/docs/legal/licensing-and-trademark/#noncommercial-and-community-web-sites) requirements.

Let us know once that is updated and we'll get this merged.

**Comment by Goyath** [comment]
Thank you, 
sorry for the misunderstanding. I changed the group name to "Meshtastic Japan Community". 
Please confirm.

**Comment by tekstrand** [comment]
@Goyath you'll need to update the name here inside of your Pull Request as well

File: docs/community/local-groups.mdx [hunk 1]
```diff
@@ -173,3 +173,6 @@ us on [Discord](https://discord.com/invite/ktMAKGBnBs) to add your group.
 
 ### Wisconsin
 - [Meshconsin](https://meshconsin.org)
+
+### Japan
+- [Meshtastic Japan](https://www.facebook.com/share/g/BQCGxZhw9SxFQja8/?mibextid=K35XfP)
```
Comment (pdxlocations) [inline docs/community/local-groups.mdx:178]: Hello, could you please change the listing here to match your group, "Meshtastic Japan Community"

I would also request that you use the M-Powered Logo, or an original design, on the facebook group.

Thank you and I look forward to adding your group.

**Review by pdxlocations — CHANGES_REQUESTED** [review]
Group listing and logo need adjustments to follow Trademark Guidelines:
https://www.facebook.com/share/g/BQCGxZhw9SxFQja8/?mibextid=K35XfP

**Comment by pdxlocations** [comment]
@tekstrand @rcarteraz wowzers I somehow missed all your comments on this. Sorry to toe-step!

File: docs/community/local-groups.mdx (hunk 1 above)
Comment (Goyath) [reply docs/community/local-groups.mdx:178]: I changed the name. Is it good?

Comment (Goyath) [reply docs/community/local-groups.mdx:178]: Does the logo need to be smaller?

Comment (pdxlocations) [reply docs/community/local-groups.mdx:178]: > I changed the name. Is it good?

The link here still says "Meshtastic Japan"

Comment (pdxlocations) [reply docs/community/local-groups.mdx:178]: > Does the logo need to be smaller?

It's not so much the size as it needs to be "secondary" to your own identity as a club, or you may wish to use the M-Powered Logo:
https://github.com/meshtastic/design/tree/master/Meshtastic%20Powered%20Logo

Comment (Goyath) [reply docs/community/local-groups.mdx:178]: https://github.com/meshtastic/meshtastic/compare/master...Goyath:patch-7?expand=1
Sorry, I'm not familiar with this, I may have rewritten it somewhere else. Please check.

**Comment by pdxlocations** [comment]
closed in favor of #1413

**Pull request closed without merging**